app/src/server/api/external/v1Api.router.ts

```diff
@@ -115,62 +103,89 @@ export const v1ApiRouter = createOpenApiRouter({
           : chatCompletionInputReqPayload.parse(input);
 
       const modelSlug = inputPayload.model.replace("openpipe:", "");
-      const fineTune = await prisma.fineTune.findUnique({
-        where: { slug: modelSlug },
-      });
-
-      if (!fineTune) {
-        throw new TRPCError({ message: "The model does not exist", code: "NOT_FOUND" });
-      }
+      const fineTune =
+        (await prisma.fineTune.findUnique({
+          where: { slug: modelSlug },
+        })) ?? undefined;
```

Comment:
What's the `?? undefined` for?

Reply:
Setting it to undefined makes our calls to `recordUsage` a little cleaner. Hopefully the refactor clarified its purpose.